src/Knp/DictionaryBundle/Twig/DictionaryExtension.php

```diff
@@ -25,7 +25,7 @@ public function __construct(DictionaryRegistry $dictionaries)
     public function getFunctions()
     {
         return array(
-            'dictionary' => new \Twig_Function_Method($this, 'getDictionary'),
+            new \Twig_SimpleFunction('dictionary', [$this, 'getDictionary']),
```

Comment:
The current lowest version of php is php5.3, so you have to use long array syntax for the moment :smile: